Gems/ScriptCanvas/Code/Editor/View/Windows/Tools/UpgradeTool/UpgradeTool.cpp

```diff
@@ -583,7 +554,7 @@ namespace ScriptCanvasEditor
 
     void UpgradeTool::SaveLog()
     {
-        AZStd::string outputFileName = AZStd::string::format("@devroot@/ScriptCanvasUpgradeReport.html");
+        AZStd::string outputFileName = AZStd::string::format("@engroot@/ScriptCanvasUpgradeReport.html");
```

Comment:
Should be `@projectroot@`, it looks like a file that will be written to and we don't want to write to engine root.

Reply:
Actually it should be @log@